src/AspNetCore/README.md

```diff
@@ -4,3 +4,14 @@
 > ./run.sh
 > browse to http://localhost:3000/ui/playground
 ```
+
+If you like to try GraphQL compile using [NativeAOT](https://github.com/dotnet/runtimelab/issues/248) which should [happens](https://github.com/dotnet/runtime/issues/61231) in .NET 7
```

Comment:
```suggestion
If you like to try GraphQL compiled using [NativeAOT](https://github.com/dotnet/runtimelab/issues/248) which should [happen](https://github.com/dotnet/runtime/issues/61231) in .NET 7
```

Reply:
@kant2002 One suggestion left. If no other changes from you left then I'll merge.